src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    private double side;
```

Comment:
The field 'side' should be declared before any methods (and constructors) and after the class declaration. Also, think about which variables should be local in the method and which should be class-level. In this case, 'side' is a property of the Square and should be a class-level variable.